src/components/PoolOverview.tsx

```diff
@@ -44,56 +44,76 @@ function PoolOverview({
     }),
   }
 
+  const hasShare = parseFloat(formattedData.userBalanceUSD) ? true : false
+
   return (
     <div className="poolOverview">
-      <Link to={to}>
-        <div className="table">
-          <h4 className="title">{formattedData.name}</h4>
-          <div className="left">
-            <span style={{ marginRight: "8px" }}>[</span>
-            {formattedData.tokens.map((token) => (
-              <div className="token" key={token.symbol}>
-                <img alt="icon" src={token.icon} />
-                <span>{token.name}</span>
-              </div>
-            ))}
-            <span style={{ marginLeft: "-8px" }}>]</span>
+      <div className="left">
+        <h4 className="title">{formattedData.name}</h4>
+        {hasShare && (
+          <div className="balance">
+            <span>{t("balance")}: </span>
+            <span>{`$${formattedData.userBalanceUSD}`}</span>
           </div>
+        )}
+        <div className="tokens">
+          <span style={{ marginRight: "8px" }}>[</span>
+          {formattedData.tokens.map((token) => (
+            <div className="token" key={token.symbol}>
+              <img alt="icon" src={token.icon} />
+              <span>{token.name}</span>
+            </div>
+          ))}
```

Comment:
nit: can destructure here to save a few characters
```js
          {formattedData.tokens.map(({symbol, icon, name}) => (
            <div className="token" key={symbol}>
              <img alt="icon" src={icon} />
              <span>{name}</span>
            </div>
          ))}
```
ref: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment